### PR TITLE
SMA 294 features/chain-integration

### PR DESCRIPTION
**Pull request by AmanRaj1608**
# Summary
Add Chilliz and astar

Related Issue: # (issue number)
SMA 294

## Change Type

- [ ] Bug Fix
- [ ] Refactor
- [x] New Feature
- [ ] Breaking Change
- [ ] Documentation Update
- [ ] Performance Improvement
- [ ] Other

### CODE REVIEW

File: packages/core-types/src/ChainsTypes.ts [hunk 1]
```diff
@@ -1,31 +1,3 @@
-export enum ChainNames {
```
Comment (livingrockrises) [inline packages/core-types/src/ChainsTypes.ts:1]: why are we removing?

**Review by livingrockrises — COMMENTED** [review]
done

File: packages/core-types/src/ChainsTypes.ts (hunk 1 above)
Comment (livingrockrises) [reply packages/core-types/src/ChainsTypes.ts:1]: @AmanRaj1608

Comment (livingrockrises) [reply packages/core-types/src/ChainsTypes.ts:1]: cool

**Review by livingrockrises — APPROVED** [review]

**Review by arcticfloyd1984 — APPROVED** [review]

**Pull request merged**